nginx.conf

```diff
@@ -0,0 +1,15 @@
+server {
+    listen       80;
+    server_name  localhost;
+
+    location / {
+        root   /usr/share/nginx/html;
+        index  index.html index.htm;
+        try_files $uri /index.html;
+    }
+
```

Comment:
@marcelosalloum copied from https://github.com/stellar/pasapeso-admin-frontend/blob/master/nginx.conf
without the env-config part.